public/content/general.tsx

```
@@ -1,94 +1,143 @@
+
+// WINDOW -> REDIRECT
+export async function redirect(target: string): void {
+    await view(target);
+    history.pushState(null, '', target);
+}
+
+// WINDOW -> MANAGER
+// @ts-nocheck
+async function view(target: string): void {
+    switch (SUG.VWD[0]) {
+        case "_": window._.remove();
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)